README.md

```diff
@@ -208,7 +208,31 @@ At a minimum, all Roaring implementations should be able to parse the two files
 
 ## Unsigned integers in Java
 
-
 Java lacks native unsigned integers, but integers are still considered to be unsigned within Roaring, and ordered  according to  ``Integer.compareUnsigned``. 
 
 For integers in [0,2147483647], the unsigned and signed integers are undistinguisable since Java relies a 32-bit two's complement format for its ``int`` type.
+
+# Extention for 64B implementations
+
+Some Roaring bitmap implementations may offer a 64-bits implementation. This section proposes a portable format, compatible with some (but not all) 
+64-bits implementations. This format is naturally compatible with implementations base on a conventional
+red-black-tree. The keys would be 32-bit integers representing the most significant 32~bits of elements whereas the values of the tree are 32-bit Roaring 
+bitmaps. The 32-bit Roaring bitmaps represent the least significant bits of a set of elements.
```

Comment:
For 63-bit numbers in `[0, 9223372036854775807]`, there is no ambiguity. But there is in general.

Reply:
Unclear to me what is expected from myself on this feedback.

Reply:
None. I am just stressing that if the keys have to be in order, then we have to worry about whether it is unsigned or signed order. This wasn't a remark I expected you to answer specifically.

Reply:
Go expects strict ordering of buckets (https://github.com/RoaringBitmap/roaring/blob/master/roaring64/roaring64.go#L114).
C seems to handle mis-ordering (https://github.com/RoaringBitmap/CRoaring/blob/master/cpp/roaring64map.hh#L1097)
Java handle mis-ordering (https://github.com/RoaringBitmap/RoaringBitmap/blob/master/RoaringBitmap/src/main/java/org/roaringbitmap/longlong/Roaring64NavigableMap.java#L1233).

But ordering would simplify/enable memory-mapping.